charts/quantum-serverless/values.yaml

```diff
@@ -131,52 +127,21 @@ kuberay-operator:
     pullPolicy: IfNotPresent
 
 # ===================
-# Keycloak
+# PostgreSQL
 # ===================
 
-keycloakEnable: true
-gatewayClientSecret: GATEWAYSECRET-CHANGEME
-grafanaClientSecret: GRAFANASECRET-CHANGEME
-keycloakUserID: user
-keycloakUserPassword: password123
-keycloak:
-  nameOverride: "keycloak"
-  fullnameOverride: "keycloak"
-
-  logging:
-    level: DEBUG
-  service:
-    type: ClusterIP
-    ports:
-      http: 31059
+postgresqlEnable: true
+postgresql:
   auth:
-    adminUser: admin
-    adminPassword: passw0rd
-  extraVolumes:
-    - name: realm
-      configMap:
-        name: keycloakrealm
-  extraVolumeMounts:
-    - name: realm
-      mountPath: /opt/bitnami/keycloak/data/import
-  extraStartupArgs: "--import-realm"
-
-  postgresql:
-    nameOverride: "postgresql"
-    fullnameOverride: "postgresql"
-    primary:
-      initdb:
-        scripts:
-          create_db.sh: |
-            #!/bin/bash
-            PGUSER=$POSTGRES_USER PGPASSWORD=$POSTGRES_PASSWORD createdb -w "repository_db"
-            PGUSER=$POSTGRES_USER PGPASSWORD=$POSTGRES_PASSWORD createdb -w "gateway_db"
```

Comment:
Do we still need to initialize the database?

Reply:
good catch, I don't think so honestly

Reply:
I tried re-adding the init bits, but that didn't help